project_event/views/project_task_view.xml

```diff
@@ -215,6 +218,7 @@
                 <field name="parent_id" string="Activity"/>
                 <field name="date_start"/>
                 <field name="date_end"/>
+                <field name="asterisk_validate_record"/>
```

Comment:
This should go at the end, i.e after  <field name="task_state"/> column

Reply:
@eilst : It is based on task description